src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -0,0 +1,533 @@
+<template>
+  <div>
+    <confirm-dialog ref="reassignCPDialog" attach="#addEditPersonContainer"/>
+    <v-expand-transition id="addEditPersonContainer">
+      <ul class="list add-person">
+        <li class="add-person-container">
+          <div class="meta-container">
+            <label class="add-person-header" v-if="isPerson">
+              <span v-if="activeIndex===-1">Add Person</span>
+              <span v-else>Edit Person</span>
+            </label>
+            <label class="add-person-header" v-if="isOrg">
+              <span v-if="activeIndex===-1">Add Corporation or Firm</span>
+              <span v-else>Edit Corporation or Firm</span>
+            </label>
+            <div class="meta-container__inner">
+              <v-form
+                ref="addPersonOrgForm"
+                class="appoint-form"
+                v-model="addPersonOrgFormValid"
+                v-on:submit.prevent="addPerson">
+                <label class="sub-header" v-if="isPerson">Person's Name</label>
+                <label class="sub-header" v-if="isOrg">Corporation or Firm Name</label>
+                <div class="form__row three-column" v-if="isPerson">
+                  <v-text-field
+                    filled
+                    class="item"
+                    label="First Name"
+                    id="person__first-name"
+                    v-model="orgPerson.firstName"
+                    :rules="firstNameRules"
+                  />
+                  <v-text-field
+                    filled
+                    class="item"
+                    label="Middle Name"
+                    id="person__middle-name"
+                    v-model="orgPerson.middleName"
+                    :rules="middleNameRules"
+                  />
+                  <v-text-field
+                    filled
+                    class="item"
+                    label="Last Name"
+                    id="person__last-name"
+                    v-model="orgPerson.lastName"
+                    :rules="lastNameRules"
+                  />
+                </div>
+                <div v-if="isOrg" class="org-name-container">
+                  <v-text-field
+                    filled
+                    class="item"
+                    label="Full Legal Corporation or Firm Name"
+                    id="firm-name"
+                    v-model="orgPerson.orgName"
+                    :rules="orgNameRules"
+                  />
+                </div>
+                <label class="sub-header">Roles</label>
+                <v-row>
+                  <v-col cols="4" v-if="isPerson">
+                    <v-checkbox v-model="isCompletingParty" label="Completing Party"
+                    :disabled="isRoleLocked(Roles.COMPLETING_PARTY)"
+                    v-bind:class="{'highlightedRole': isRoleLocked(Roles.COMPLETING_PARTY)}"
```

Comment:
"highlightedRole" makes checkbox hard to read. If possible, should be white text on green background. See mockup.

Also, if possible, checkboxes should be "vertically centered"... check the vertical margins/padding to even this out.

Reply:
Check the styles of the other checkboxes here. According to the mockup they should have a gray background if not checked or a green background if checked.

Reply:
Looking into this. It's definitely a result of some inherited css from Vuetify, should be able to override them! Working on er

Reply:
Spacing has been fixed but currently WIP on overriding the text color.

Reply:
Was this because of style typo? Fixed now?

Reply:
Fixed now! 👍